specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json

```diff
@@ -1363,7 +1388,7 @@
       "schema": {
         "$ref": "#/definitions/CreateManagementGroupRequest"
       },
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "client"
```

Comment:
>client [](start = 34, length = 6)

why is this parameter changed from method location to client? By doing this change you require developers to set global parameter on the client instance and not on the method parameters.